api/app/src/shared/external.ts

```diff
@@ -11,18 +11,17 @@ export const getDocumentPrimaryId = (document: Document): string => {
   return encodeExternalId(id);
 };
 
-const NON_HL7_SUFFIX = "-M-";
+const HL7_SUFFIX = ".";
 
 export function encodeExternalId(decodedId: string): string {
   const hasHl7Prefix = decodedId.includes(HL7OID);
-  const idToEncode = hasHl7Prefix ? decodedId.replace(HL7OID, "") : NON_HL7_SUFFIX + decodedId;
+  const idToEncode = hasHl7Prefix ? decodedId.replace(HL7OID, "") : decodedId;
```

Comment:
Should this be :

`const idToEncode = hasHl7Prefix ? decodedId.replace(HL7OID, HL7_SUFFIX) : decodedId;` ?

Reply:
🧠